### DISCUSSION TITLE
editOnClick style issue when content is empty

### DISCUSSION THREAD

**Discussion by jwhoami** [Power-Components/livewire-powergrid#511] [Q&A]
Hi, I have an created a cell action with editOnClick. This field has a required validation.
So if the field has a value and I edit it and clear the value and press enter, the content is empty and is reduced to a line like this...

![image](https://user-images.githubusercontent.com/74318608/179454860-8138bd3c-b65b-4a17-95cf-304b951b41c9.png)

I have to click on the thin line above the error to access the input element. Since the line is so thin, it hard to do so.

Is there a way that if the field is empty, it has a default height like this

![image](https://user-images.githubusercontent.com/74318608/179455200-a18a443e-f0bf-4925-b7f2-0ed5e6545313.png)

Its not clear on the second image, but now I can click above the line and access the input element. To get this working, I had to
to set the style in the editable.blade.php file like this...

`    <div x-html="content"
         style="border-bottom: dotted 1px; cursor: pointer; height:24px"
         x-show="!editable"
         x-on:click="editable = true"
    ></div>
`
The solution is hacky, so I wanted to know if there is a better way.

**Comment by Reefcrazed** [comment]
What I have found, this normally means you did not go through the entire installation. Your themes essentially are not working, go back through the installation, pay attention to the configuration section and make sure to not skip that part
